### PR TITLE
CAPTSTONE

### PR DESCRIPTION
**Pull request by Mutalenic**
In this capstone I implemented the project using;

- HTML tags
- CSS 
- JavaScript
- JavaScript DOM manipulation
- Flexbox 
- Grid
- Media queries
- Used GitHub Pages to deploy web pages
- I made a video explaining the layout of my project web page
Here is the link to my video presentation : https://www.loom.com/share/21956dfbd1a54c4394456f83d2a5eb93

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        </div>
+      </div>
+      <hr class="sectionline" />
+    </section>
+
+    <section>
+      <div>
+        <h3 class="pastEvents">See the past HIR Africa Conference</h3>
+        <hr class="pastEventsLine" />
+      </div>
```
Comment (ShoiraTa) [inline about.html:127]: - [ ] In mobile view the images and the headings of this section should be centered, currently, in your page they are on the left side, please kindly fix this:
- 
Behance design  | Your design
------------- | -------------
image  | image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        <span class="microsoft">Microsoft</span>
+      </div>
+    </section>
+    <footer>
+      <div class="abtfooter">
+        <img src="images/logo.png" alt="eventlogo" class="eventlogo" />
+        <div>
+          <p>&copy;2022 HIR Nkwashi</p>
+          <p>
+            HIR-Nkwashi is a place East of Lusaka city that serves as a center
```
Comment (ShoiraTa) [inline about.html:164]: - [ ] In mobile view there is some white space between the footer and upper section, please delete that space and make the mobile footer background white as provided in behance design: 

Behance  | Yours
------------- | -------------
image  | image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+      gtag("js", new Date());
+
+      gtag("config", "G-CLN7DN2Y13");
+    </script>
+  </head>
+  <body>
+    <main>
+      <nav class="navbar">
+        <ul class="navbarlist">
+          <li><i class="fab fa-facebook-f"></i></li>
```
Comment (ShoiraTa) [inline index.html:43]: - [ ] OPTIONAL: Users can scroll the page when the mobile `navbar` is open, this is not a good practice, please consider adding `overflow: hidden` to body while mobile `nav` is open.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+            <p class="mainProgramtext">
+              Create opportunities to freely network with new normal business
+              around Africa.
+            </p>
+          </div>
+        </div>
+
+        <button class="joinbtn" type="button">
+          Join HIR Africa Conference 2022
+        </button>
```
Comment (ShoiraTa) [inline index.html:161]: - [ ] OPTIONAL: Please consider centering this button:
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "G-CLN7DN2Y13");
+    </script>
+  </head>
+  <body>
+    <main>
+      <nav class="navbar">
```
Comment (ShoiraTa) [inline about.html:39]: - [ ] The `<main>`  element represents the dominant content of the <body> of a document. The main content area consists of content that is directly related to or expands upon the central topic of a document, or the central functionality of an application. But in your case `main`  is wraping only the `navbar`, I suggest you to replace `<main>` to `<header>`

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @Mutalenic ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] The live link provided in README is taking to  readme(https://mutalenic.github.io/FirstCapstone/)  page. Within the reuirements for this milestone, please deploy your page in  GitHub Pages and make sure that all pages, links, and images work as expected.

- [ ] In readme, please give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in "Acknowledgements". 

- [ ] OPTIONAL: I strongly suggest you to update readme file, currently it looks confusing, for example, secondary headings font size is bigger than primary, in built with section you have two empty listings, then suddenly a screenshot:
image 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Mutalenic 

Thank you for implementing some of the recommended changes by the previous reviewer, however, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the inline comments under the review for sections to improve._

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Mutalenic** [comment]
> Hi @Mutalenic
> 
> Thank you for implementing some of the recommended changes by the previous reviewer, however, there are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Required Changes ♻️
> _Check the inline comments under the review for sections to improve._
> 
> Optional suggestions Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
> 
> Cheers and Happy coding!
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

noted

**Review by elmar8287 — COMMENTED** [review]
Hi @Mutalenic , 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**